contracts/eosio.system/src/rex.cpp

```diff
@@ -359,7 +245,7 @@ namespace eosiosystem {
    }
 
    /**
-    * @brief Moves a specified amount of REX to savings bucket 
+    * @brief Moves a specified amount of REX to savings bucket.
```

Comment:
Can you move this to the HPP like the rest, mvtosavings is an action like the rest.